decaylanguage/dec/dec.py

```diff
@@ -435,10 +432,10 @@ def _add_decays_to_be_copied(self):
                 copied_decay = match.__deepcopy__(None)
                 copied_decay.children[0].children[0].value = decay2copy
                 copied_decays.append(copied_decay)
-            except:
+            except Exception:
```

Comment:
No bare excepts! These might be able to be tightened up further, but at least now they don't catch memory errors, control C, etc.

Reply:
Good catch. This was indeed overlooked.